pgmpy/estimators/MLE.py

```diff
@@ -62,6 +62,7 @@ def get_parameters(self):
             if not parents:
                 state_counts = self.data.ix[:, node].value_counts()
                 state_counts = state_counts.reindex(sorted(state_counts.index))
+                state_counts = state_counts.astype(np.int, copy=False)
```

Comment:
@pheman I think a better way of doing this would be to check for the type when defining `self.data` in `BaseEstimator`.